src/charts/chart.js

```diff
@@ -142,6 +142,8 @@ export default function Chart (_container) {
   }
 
   const cache = {
+    data: null,
+    config,
```

Comment:
Raw `data` and `config` that comes from the user are cached here.

Reply:
why cache these?

Reply:
To keep a reference to the raw data and config, so we keep a clear separation between user input and internal data/config.